magicctapipe/scripts/lst1_magic/README.md

```diff
@@ -0,0 +1,234 @@
+The output in the terminal will be something like this:
+```
+***** Generating merge_MAGIC bashscripts...  
+***** Running merge_hdf_files.py in the MAGIC data files...  
+Process name: merging_Crab  
+To check the jobs submitted to the cluster, type: squeue -n merging_Crab
+This process will take about 10 to 30 min to run.
+```
+
+This script will merge the MAGIC data files in the following order:
+- MAGIC subruns are merged into single runs.
+- MAGIC I and II runs are merged (only if both telescopes are used, of course).
+- All runs in specific nights are merged, such that in the end we have only one datafile per night.
+
+### Coincident events and stereo parameters on DL1
+
+To find coincident events between MAGIC and LST, starting from DL1 data, we run the following command in the working directory:
+
+> $ coincident_events (-c config_general.yaml)
+
+This script creates the file config_coincidence.yaml containing the telescope IDs and the following parameters:
+```
+event_coincidence:
+    timestamp_type_lst: "dragon_time"  # select "dragon_time", "tib_time" or "ucts_time"
+    pre_offset_search: true
+    n_pre_offset_search_events: 100
+    window_half_width: "300 ns"
+    time_offset:
+        start: "-10 us"
+        stop: "0 us
+```
+
+Then, for each NSB level, it reads the corresponding txt file generated by the second script and, taking into account the day (and, as a consequence the MAGIC period of the joint MAGIC runs) links the LST data files for these runs to the output directory [...]DL1Coincident/ST{}/NSB{}, and runs the script lst1_magic_event_coincidence.py in all of them.
+
+Once it is done, we add stereo parameters to the MAGIC+LST coincident DL1 files by running:
+
+> $ stereo_events (-c config_general.yaml)
+
+This script creates the file config_stereo.yaml with the following parameters:
+```
+stereo_reco:
+    quality_cuts: "(intensity > 50) & (width > 0)"
+    theta_uplim: "6 arcmin"
+```
+
+It then creates the output directories for the DL1 with stereo parameters [...]DL1CoincidentStereo/ST{}/NSB{}, and then runs the script lst1_magic_stereo_reco.py in all of the coincident DL1 files. The stereo DL1 files are then saved in these directories.
+
+### Random forest and DL1 to DL2
+
+TBD.
+
+### Instrument response function and DL3
+
+TBD.
+
+## High-level analysis
+
+Since the DL3 may have only a few MBs, it is typically convenient to download it to your own computer at this point. It will be necessary to have astropy and gammapy (version > 0.20) installed before proceeding. 
+
+We prepared a [Jupyter Notebook](https://github.com/ranieremenezes/magic-cta-pipe/blob/master/magicctapipe/scripts/lst1_magic/SED_and_LC_from_DL3.ipynb) that quickly creates a counts map, a significance curve, an SED, and a light curve. You can give it a try. 
```

Comment:
why not committing the notebook to the main repository? (into the notebooks driectory, and not the scripts one)

Reply:
actually this notebook does not seem to exist in the current branch. We only have to remove this sentence from the readme

Reply:
removed